roswallp.txt

```diff
@@ -0,0 +1,15 @@
+[Section]
```

Comment:
Is this file the one that needs to be used in the reactos/rapps-db sub-project?

Reply:
Yes with modification of `???`.